xds/internal/xdsclient/xdsresource/filter_chain.go

```diff
@@ -66,6 +68,8 @@ type FilterChain struct {
 	//
 	// Exactly one of RouteConfigName and InlineRouteConfig is set.
 	InlineRouteConfig *RouteConfigUpdate
+	// RC is the routing configuration for this filter chain (LDS + RDS).
+	RC *unsafe.Pointer // *(RoutingConfiguration)
```

Comment:
Optional: Maybe a more descriptive field name is called for, given that this field is used in other packages?

Reply:
Good point. As you told me to switch the type name, I also switched RC to UsableRoutingConfiguration.